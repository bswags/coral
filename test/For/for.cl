#cordbconn
server="server";
user="server";
password="server";
port="server";
DBName="server";
type="server";
#enddbconn

#cordb
#enddb

void main() {

	int i = 0;
	for(i = 0; i < 10; i++){
		i = i*2;
	}
	printf("%d",i);

}