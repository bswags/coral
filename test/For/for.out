Traceback (most recent call last):
  File "For/for.clx", line 6, in <module>
    setServer("server")
NameError: name 'setServer' is not defined
